gravity-forms/gw-none-of-the-above-checkbox.js

```diff
@@ -26,7 +26,7 @@ $( '.gw-none-of-the-above' ).each( function() {
 	var $others = $field.find( 'input' ).not( $last );
 
 	// If "None of the Above" choice is checked by default.
-	if ( $( last ).prop( 'checked' ) ) {
+	if ( $( $last ).prop( 'checked' ) ) {
```

Comment:
This could technically be the following since `$last` is already a jQuery object. I'll still approve, however.

```suggestion
	if ( $last.prop( 'checked' ) ) {
```